deps/pcre.mk

```diff
@@ -55,7 +55,7 @@ fastcheck-pcre: check-pcre
 check-pcre: $(BUILDDIR)/pcre2-$(PCRE_VER)/build-checked
 
 else # USE_BINARYBUILDER_PCRE
-PCRE_BB_URL_BASE := https://github.com/JuliaPackaging/Yggdrasil/releases/download/PCRE2-v$(PCRE_VER)-$(PCRE_BB_REL)
+PCRE_BB_URL_BASE := https://github.com/JuliaBinaryWrappers/PCRE2_jll.jl/releases/download/PCRE2-v$(PCRE_VER).0+$(PCRE_BB_REL)
```

Comment:
Should this `.0` be moved to `PCRE_VER` in `Make.inc` instead?

Reply:
Then it wouldn't work when downloading the sources instead:
https://github.com/JuliaLang/julia/blob/master/deps/pcre.mk#L9
(I'm not sure why PCRE2_jll insist on adding another number suffix, it really is just 10.34, https://ftp.pcre.org/pub/pcre/)

Reply:
> (I'm not sure why PCRE2_jll insist on adding another number suffix, it really is just 10.34, ftp.pcre.org/pub/pcre)

Because JLL packages use Julia's `VersionNumber` representation in string interpolation:
```julia
julia> "$(v"10.34")"
"10.34.0"
```